Styling

peterdgreat opened the pull request:
## Basic styling added
- Background color added to the body
- Background color changed for list items
- Border removed from all items
- All text colors changed to #fff

peterdgreat committed:
Little styling added

peterdgreat committed:
merging...

peterdgreat committed:
Readme Updated

- Screenshot added

peterdgreat committed:
Button color changed and screenshot updated

henatan99 commented on src/style.css:
```diff
@@ -33,3 +42,15 @@ input {
+
+.wrapper {
+  display: flex;
+  justify-content: center;
```
- [ ] This styling hides the top contents when the page content is populated beyond the view height. Kindly remove the `justify-content` property or change its value. 

![Screenshot from 2021-10-08 15-32-41](https://user-images.githubusercontent.com/64894927/136558669-21e2549d-303f-4874-ac12-14ea421e4dfb.png)

henatan99 reviewed (changes requested):
Hi @peterdgreat , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

peterdgreat committed:
Update wrapper height

- change from 100vh to 100% to prevent the weird overlay after the view height has been exceeded

peterdgreat commented on src/index.html:
```diff
@@ -11,9 +11,13 @@
-<header>
-    <h1>Leaderboard</h1>
-</header>
+    <header class="mt-5">
+        <h1 class="text-center">Leaderboard</h1>
+    </header>
+    <div class="wrapper">
+
+    
+
```
Ohh thanks George, will work on those
